partner_time_to_pay/readme/INSTALL.rst

```diff
@@ -0,0 +1,7 @@
+Your preferred way to install addons will work with Partner Time to Pay.
```

Comment:
Obsolete since there are no specific instructions

Reply:
Yes, @alexey-pelykh, here no need to put other external configuration or specific instructions.

it's simple to install and use.

Reply:
But do we need this instruction at all? What module-specific steps it contain? Or it's just "install like any other module"